pkgs/build-support/binutils-wrapper/ld-wrapper.sh

```diff
@@ -67,8 +67,16 @@ declare -a libDirs
 declare -A libs
 relocatable=
 
-# Find all -L... switches for rpath, and relocatable flags for build id.
-if [ "$NIX_@infixSalt@_DONT_SET_RPATH" != 1 ] || [ "$NIX_@infixSalt@_SET_BUILD_ID" = 1 ]; then
+# Two tasks:
+#
+#   1. Find all -L... switches for rpath, and relocatable flags for build id.
```

Comment:
These cases are not related, so if you enumerate them, they should be 1 and 2.

Reply:
True. Good catch.

Reply:
Done